beater/sipcmbeat.go

```diff
@@ -612,8 +620,7 @@ func (bt *Sipcmbeat) publishEv(srcEv *calltr.EventData) {
 	addFields(event.Fields, "dbg.last_status", ed.LastStatus)
 	addFields(event.Fields, "dbg.msg_trace", ed.LastMsgs.String())
 
-	addFields(event.Fields, "fflags", fFlags)
-
+	addFields(event.Fields, "encrypt", strconv.Itoa(int(fFlags))+"|"+bt.validator.Compute())
```

Comment:
the bt.validator.Compute() part needs to be added only if encryption is enabled.